docs/ABOUT.md

```diff
@@ -0,0 +1,14 @@
+Javascript is a scripting language used to provide dynamic and interactive content on webpages. It does not have class-oriented inheritance, but it does have prototype-oriented inheritance. Also, server side JS allows to use same language on server and client besides being fast. JavaScript provides benefits like :- 
```

Comment:
The third sentence seems to have a few issues.  Should read perhaps "allows one to use the same language on the server and client.  Besides being fast, JavaScript provides..."